tests/cli/doc/custom_logo/inputs/file.wdl

```diff
@@ -0,0 +1,7 @@
+## Rust logo downloaded from https://github.com/rust-lang/rust-artwork/blob/bf0b3272f9ba8d22f7fd45e408496d05621b3b5c/2017-RustConf/Rust_Lucy%20Art_B.svg
+##
+## License: https://creativecommons.org/licenses/by/4.0/
```

Comment:
I think this particular piece is under this license: https://creativecommons.org/licenses/by-nc-nd/4.0/. This is according to this section of the readme: https://github.com/rust-lang/rust-artwork/blob/bf0b3272f9ba8d22f7fd45e408496d05621b3b5c/README.md#licensing-terms-for-the-rustconf-artwork.

Let me just make us an example that we own.